packages/mrm/src/requirex.js

```diff
@@ -0,0 +1,33 @@
+const npx = require('libnpx');
+const path = require('path');
+
+async function requirex_(packageName) {
+	const NPM_PATH = path.join(
+		__dirname,
+		'..',
+		'node_modules',
+		'npm',
+		'bin',
+		'npm-cli.js'
+	);
+	const results = await npx._ensurePackages(packageName, { npm: NPM_PATH });
+	const packagePath = path.join(
+		results.prefix,
+		'lib',
+		'node_modules',
+		packageName
+	);
+	return packagePath;
+}
+
+async function requirex(packageName) {
+	const packagePath = await requirex_(packageName);
+	return require(packagePath);
+}
+
+requirex.resolve = async function(packageName) {
+	const packagePath = await requirex_(packageName);
+	return require.resolve(packagePath);
+};
+
+module.exports = requirex;
```

Comment:
The original problem is that we may not have the given task or preset installed when the `mrm` command runs. The solution for presets was to interactively install that preset globally. But globally installing packages is not great for the reason I mentioned in the issue.

I found inspiration from the `npx` command which installs packages on-the-fly but without the downsides of globally installing them. I checked the source code and it seemed reusable. This is a tiny wrapper around `ensurePackages` from `libnpx` for this purpose.

Reply:
This was autocomplete ;-( I was asking about the name — why `requirex`?

Reply:
Ah, because it uses `npx` under the hood, but I'm not too tied to the name. I'm also considering extracting it to a separate package but it's not a priority.

Reply:
Shouldn't it be `requireUsingNpx` in this case? I think I'm still missing something ;-F

Reply:
I can rename, I was probably thinking of a cool package name. :D

Reply:
Done!